### DISCUSSION TITLE
Serving UI on NAS, indexing on PC

### DISCUSSION THREAD

**Discussion by anemes** [photoprism/photoprism#4532] [Q&A]
Hi, I would love to host photoprism on an small arm64 NAS, however it isn't powerful enough to do the photo import / indexing. I am running it in a container and was wondering if anyone has set this up before. I am thinking.

1. Mysql on arm64 NAS open to PC access
2. Storage on NAS share (where images are)
3. Library import using a more powerful PC and index to NAS storage / mysql
4. Serve with NAS

**Comment by DJKeterson** [comment]
Okay so this should be possible by doing the following:

- set import folder on your machine with good CPU and flash storage
- set originals folder on your nas
- dump all your photos / folders in the import folder
- run the import function on the library menu option in PP web UI

Does that make sense to you @anemes ?
